instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import Comment from '../CommentSection/Comment';
+import PostInteraction from '../PostInteraction/PostInteraction';
+import propTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import '../CommentSection/commentsection.css';
+
+class CommentSection extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      comments: props.comments,
+      newComment: '',
+      likes: this.props.likes
+    };
+  }
+
+  addNewComment = event => {
+    event.preventDefault();
+    this.setState({
+      comments: [
+        ...this.state.comments,
+        { text: this.state.newComment, username: 'Brandon' }
```

Comment:
now that you have the username on localStorage. It would be nice if the login matched the username of the person logged in.